Rate limit section (#370)
permitio/docs: docs/api/api-with-cli.mdx

This can be achieved with Postman or any other REST client as well as with CLI tools like curl.

> ## Rate Limiting
> 
> Permit's API is rate-limited to prevent abuse. 
> 
> The number of requests you can make is limited to 5,000 per minute.
> For facts requests (`/v2/facts/*`), the limit is 3,000 per minute.
> For schema requests (`/v2/schema/*`), the limit is 1,000 per minute.
> For bulk requests, the limit is 100 per 10 minutes.
> 
> The rate limit is per customer and can be adjusted, reach out to us if you need to adjust the rate limit.
> 
> 

Reviewer: ```suggestion
## Rate Limiting

Permit's API is rate-limited to prevent abuse. 

The number of requests you can make is limited to 5,000 overall requests per minute.
For fact requests (`/v2/facts/*`), the limit is 3,000 per minute.
For schema requests (`/v2/schema/*`), the limit is 1,000 requests per minute.
For bulk requests, the limit is 100 requests per 10 minutes.

The rate limit is adjustable per customer, reach out to us if you need to adjust the rate limit.
```
